output-data/index.ts

```diff
@@ -0,0 +1,15 @@
+import { ActualOffenceDate, AlcoholLevelMethod, Amount } from "./types/types"
+
+import actualOffenceDateData from "./data/actual-offence-date.json"
+import alcoholLevelMethodData from "./data/alcohol-level-method.json"
+import amountData from "./data/amount.json"
+
+const actualOffenceDate: ActualOffenceDate[] = actualOffenceDateData
+const alcoholLevelMethod: AlcoholLevelMethod[] = alcoholLevelMethodData
+const amount: Amount[] = amountData
+
+export default {
+  actualOffenceDate,
+  alcoholLevelMethod,
+  amount
```

Comment:
Why aren't we also importing/typing/exporting the other JSON files?

Reply:
Will do it the next slice